src/test/java/org/mastodon/mamut/clustering/ui/DendrogramPanelTest.java

```diff
@@ -146,6 +163,26 @@ public void testGetVerticalLine()
 		assertEquals( 312d, line.getX1(), 0.01 );
 	}
 
+	@Test
+	public void testExport() throws IOException
+	{
+		DendrogramPanel< String > dendrogramPanel = new DendrogramPanel<>( classification );
+		int width = 600;
+		int height = 600;
+		Image image = new BufferedImage( width, height, BufferedImage.TYPE_INT_ARGB );
+		Graphics graphics = image.getGraphics();
+		dendrogramPanel.setSize( width, height );
+		dendrogramPanel.paint( graphics );
+		File tempFile = File.createTempFile( "dendrogram", ".png" );
+		tempFile.deleteOnExit();
+		int screenResolution = 96;
+		int printResolution = 600;
+		dendrogramPanel.export( tempFile, "png", screenResolution );
+		Image test = ImageIO.read( tempFile );
+		assertEquals( width * printResolution / screenResolution, test.getWidth( null ) );
```

Comment:
How about `assertTrue( test.getWidth( null ) > 10 )`, this would make the test less confusing. The test would  ensure that a legitimate image is created, and the focus would move away from the less important resolution computation.

Reply:
Hm, I do not agree on this. I would find `assertTrue( test.getWidth( null ) > 10 )` confusing. I would ask myself, why the choice of 10? And why larger than 10. This would be confusing to me.

I agree that the computation of the expected output width within the assertEquals() is not easy to read. I changed that.

Beyond that testing whether the exported image has the expected size according to the given resolution param was intended to be part of this test.